### DISCUSSION TITLE
下载错误

### DISCUSSION THREAD

**Discussion by xdzz0906** [community/community#35570] [Other Feature Feedback, Questions, & Ideas]
下载时一直报错，无法正常运行
设备：surface laptop 3 
OS: Ubuntu 20.04

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @xdzz0906 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
